test/test-WebRtcTransport.js

```diff
@@ -337,6 +337,13 @@ test('webRtcTransport.setMaxIncomingBitrate() succeeds', async () =>
 		.toBeUndefined();
 }, 2000);
 
+test('webRtcTransport.setMaxOutgoingBitrate() succeeds', async () =>
```

Comment:
These tests are not replicated in Rust version, would be nice to have them there, should be fairly straightforward.

Reply:
Rust version the way you did it should be fine if it compiles and passes tests